csharp-intermediate-topics/ReadOnlyModifierInCSharp/ReadOnlyTests/ReadOnlyTests.cs

```diff
@@ -0,0 +1,26 @@
+using ReadOnlyModifierInCSharp;
+
+namespace ReadOnlyTests
+{
+    public class ReadOnlyTests
+    {
+        [Fact]
+        public void GivenCircleWhenRadiusSetInCtorThenAreaCalculatedUsingRadius()
+        {
+            var circle = new Circle(7.5);
+            var expectedArea = 176.71;
```

Comment:
Along with this being a const, since the Circle class is going to compute this area, we should actually perform the computation here, rather than just state a value:
`const double expectedValue = Math.PI * 7.5 * 7.5;`